cmd/gpu_plugin/README.md

```diff
@@ -313,6 +316,15 @@ The GPU plugin functionality can be verified by deploying an [OpenCL image](../.
       Warning  FailedScheduling  <unknown>  default-scheduler  0/1 nodes are available: 1 Insufficient gpu.intel.com/i915.
     ```
 
+## Use Cases for Different Modes
+
+Intel GPU-plugin supports a few different operation modes. Depending on the workloads the cluster is running, some modes make less sense than others. Below is a table that explains the differences between the modes and suggests workload types for each mode. The mode selection requires pre-though as it is cluster wide.
+
+| Mode | Sharing | Workload examples | Time critical |
+|:---- |:-------- |:------- |:------- |
+| shared-dev-num == 1 | No, 1 container per GPU | AI/ML training or an application that can fully utilize a GPU e.g. pack multiple workloads. | Yes |
+| shared-dev-num > 1 | Yes, >1 containers per GPU | Inference, media transcode, media analytics. Workloads that only require part of the GPU. See also [allocation profiles](#modes-and-configuration-options). | No |
+| shared-dev-num > 1 && resource-management | Yes and no, 1>= containers per GPU | All workloads, but requires GPU resource allocations. GPUs can be dedicated and shared based on the GPU resources like memory and millicores. Requires [GAS](https://github.com/intel/platform-aware-scheduling/tree/master/gpu-aware-scheduling). See also [fractional use](#fractional-resources-details). | Depends on the Pod spec |
```

Comment:
First state the intended workload type, and then give examples of them, for example:
* "AI/ML training or an application that can fully utilize a GPU e.g. pack multiple workloads" -> "Workloads using all GPU capacity, e.g. AI training"
* "Inference, media transcode, media analytics. Workloads that only require part of the GPU" -> "(Batch) workloads using only part of GPU resources, e.g. inference, media transcode/analytics"
* "All workloads, but requires GPU resource allocations. GPUs can be dedicated and shared based on the GPU resources like memory and millicores" -> "Any.  For best results, *all* workloads should declare their expected GPU resource usage (memory, millicores)"

What means "Time critical" depending pod spec??

Reply:
Thanks.
I rewrote "depending on pod spec" as "Depends on the requested GPU resources".